app/models/progress_report.rb

```diff
@@ -5,15 +5,30 @@ class ProgressReport < VersionedRecord
   has_many :recommendations, through: :measures
   has_many :categories, through: :recommendations
   has_one :manager, through: :indicator
+  delegate :email, to: :manager, prefix: true, allow_nil: true
+  delegate :name, to: :manager, prefix: true, allow_nil: true
 
   validates :title, presence: true
   validates :indicator_id, presence: true
 
   validate :indicator_id_must_not_change, if: [:persisted?, :indicator_id_changed?]
 
+  def self.send_all_updated_emails
+    joins(:versions)
+      .where(versions: {
+        created_at: 24.hours.ago..,
+        whodunnit: User.joins(:roles).where(roles: {name: "contributor"}).pluck(:id)
+      })
```

Comment:
```suggestion
      .where(versions: {created_at: 24.hours.ago..})
```

if we don't care about it being only contributors

Reply:
I think what we can already say is that it should NOT trigger notifications to the category manager, when the change is done by the category manager =)

Reply:
@parndt no need to limit notifications to actual contributors and draft reports. while this will be the usual case, category managers should also be notified when anyone else is creating or changing a progress report